scalanativelib/src/ScalaNativeModule.scala

```diff
@@ -214,9 +214,12 @@ trait ScalaNativeModule extends ScalaModule { outer =>
     )
   }
 
+  // Defines the native binary name
+  def nativeBinaryName: String = { this.toString }
```

Comment:
Regardless of the feature itself, this is IMHO a bad default. We have more meaningful options like `artifactName`.

Reply:
Besides, the choosen fixed default ("out") isn't arbitrary, but allows other tools to easily consume files without the need to deeply inspect the build.

Reply:
Ah sure, I could fix that.